test/integration/invalid-custom-routes/test/index.test.js

```diff
@@ -163,14 +159,6 @@ const runTests = () => {
       `\`permanent\` is not set to \`true\` or \`false\` for route {"source":"/hello","destination":"/another","permanent":"yes"}`
     )
 
-    expect(stderr).toContain(
-      `\`permanent\` is not set to \`true\` or \`false\` for route {"source":"/hello","destination":"/another","permanent":"yes"}`
-    )
-
-    expect(stderr).toContain(
-      `\`permanent\` is not set to \`true\` or \`false\` for route {"source":"/hello","destination":"/another","permanent":"yes"}`
-    )
```

Comment:
Similar as the other test, why did we change these?

Reply:
because completely the same asset is already done